app/main.py

```diff
@@ -1,2 +1,52 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.capacity = 8
+        self.threshold = int(self.capacity * 2 / 3)
+        self.hash_table = [[] for _ in range(self.capacity)]
+        self.storage = 0
+
+    def __getitem__(self, key: (int, float, str, tuple, bool)) -> list:
```

Comment:
```suggestion
    def __getitem__(self, key: (int, float, str, tuple, bool)) -> list:
```

`Key` can be class.
You should put `hashable`